istio.deps

```diff
@@ -4,7 +4,7 @@
     "name": "PROXY_REPO_SHA",
     "repoName": "proxy",
     "file": "",
-    "lastStableSHA": "ce668311b7cc8518b5cda16a8581c9eb1a3dc3cd"
+    "lastStableSHA": "6b493c7aca42e8551d7bcc50ab65288bd115d54b"
```

Comment:
This looks fine, we're just ahead of the sync from https://github.com/istio/proxy/commits/master/